workflow-templates/deploy-preview.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Quarto Deploy Preview Workflow",
+    "description": "Quarto Deploy preview starter workflow.",
+    "iconName": "octicon search",
```

Comment:
we could have any icon here, or even an svg https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization perhaps of the dasl or wilds logo?

Reply:
I'm confused about where this icon would show up but let's ship something so we can change it later.

Reply:
yep, can change later, i'll show you  if it's not clear